<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showdata</class>
 <widget class="QWidget" name="showdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>158</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);
background-color: rgba(114, 159, 207,80);}
#lineEdit{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#lineEdit_2{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#lineEdit_3{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#lineEdit_4{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#lineEdit_5{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#lineEdit_6{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>5</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>cursor x_1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>5</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>cursor x_2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>5</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>cursor y_1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>5</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>cursor y_1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>5</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>x_1-x_2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>5</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>y_1-y_2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
